detection-rules/brand_impersonation_bbb.yml

```
@@ -0,0 +1,64 @@
+name: "Brand Impersonation: The Better Business Bureau (BBB)"
+description: "Detect impersonation of the Better Business Bureau (BBB) from unsolicited sender"
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and 
+  // display name contains bbb
+  (
+    regex.icontains(strings.replace_confusables(sender.display_name),
+                    '(?:\bBBB\b|better business bureau)'
+    )
+    // or levenshtein distance similar to the full name
+    or strings.ilevenshtein(strings.replace_confusables(sender.display_name),
+                            'better business bureau'
+    ) <= 1
+  )
+  // doesn't contain a link to the legit method of responding to bbb complaints
+  and not any(body.links,
+          .href_url.domain.domain == "respond.bbb.org"
+          and .href_url.path == "/respond/"
+          and .display_url.domain.domain == "respond.bbb.org"
+          and .display_url.path == "/respond/"
+  )
+  // and the sender is not from the bbb and passes auth
+  and not (
+    (
+      sender.email.domain.root_domain in (
+        "bbb.org", 
+        "mybbb.org", 
+        "bbbcommunity.org", // official BBB site
+        "bluebbb.org", // official BBB site
+        "thebbb.org", // official BBB site        
+        "thefirstbbb.org",
+        "bbbnp.org" // unrelated domain that sends with BBB within the display name
+      )
+     or sender.email.domain.domain in (
+        "bbbwisegivingalliance.ccsend.com" // official BBB comms for events
+     )
+   )
+    and headers.auth_summary.dmarc.pass
+  )
+  
```

remove non-english messages
https://platform.sublime.security/messages/e352dd62b391379049c577739595434c4965b4db8816abda0b7f401318a91f5a?preview_id=0195a755-3e4b-7664-a9bf-3f677eacb343
```suggestion
  and ml.nlu_classifier(body.current_thread.text).language == "english"
```